### ISSUE TITLE
Related Navigation: Accessibility: Landmarks should have a unique role or role/label/title (i.e. accessible name) combination (landmark-unique)

### ISSUE THREAD

**Issue by ChristineWuerth** [alphagov/govuk-design-guide#171]
### Description

This is raised as an accessibility issue in the component guide: https://components.publishing.service.gov.uk/component-guide/related_navigation
